staging/src/k8s.io/client-go/tools/leaderelection/leaderelection.go

```diff
@@ -159,6 +159,9 @@ type LeaderElectionConfig struct {
 
 	// Name is the name of the resource lock for debugging
 	Name string
+
+	// Coordinated will use the Coordinated Leader Election feature
+	Coordinated bool
```

Comment:
```suggestion
	// Coordinated will use the Coordinated Leader Election feature.
	// WARNING: Coordinated leader election is ALPHA.
	Coordinated bool
```

Reply:
We might want to go as far as calling this `AlphaCoordinated`. Not sure we have precedence for that.